opendcs-rest-api/src/main/java/org/opendcs/odcsapi/res/ComputationResources.java

```diff
@@ -63,13 +71,32 @@ public final class ComputationResources extends OpenDcsResource
 	@Path("computationrefs")
 	@Produces(MediaType.APPLICATION_JSON)
 	@RolesAllowed({ApiConstants.ODCS_API_GUEST})
-	public Response getComputationRefs(@QueryParam("site") String site,
-			@QueryParam("algorithm") String algorithm,
-			@QueryParam("datatype") String datatype,
-			@QueryParam("group") String group,
-			@QueryParam("process") String process,
-			@QueryParam("enabled") Boolean enabled,
-			@QueryParam("interval") String interval)
+	@Operation(
+			summary = "Retrieve Computation References",
+			description = "Example:  \n\n    http://localhost:8080/odcsapi/computationrefs",
+			tags = {"REST - Computation Methods"},
+			responses = {
+					@ApiResponse(responseCode = "200", description = "Success",
+							content = @Content(mediaType = MediaType.APPLICATION_JSON,
+									array = @ArraySchema(schema = @Schema(implementation = ApiComputationRef.class)))),
+					@ApiResponse(responseCode = "404", description = "No computations found matching the filter criteria"),
+					@ApiResponse(responseCode = "500", description = "Internal Server Error")
+			}
+	)
+	public Response getComputationRefs(@Parameter(schema = @Schema(implementation = String.class))
+		@QueryParam("site") String site,
```

Comment:
The formatting here just seems wonky, would it be better to group these on one line instead of the nesting?